vktrace/src/vktrace_layer/vktrace_lib_trace.cpp

```diff
@@ -175,6 +265,8 @@ VKTRACER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL __HOOKED_vkMapMemory(
     VkFlags flags,
     void** ppData)
 {
+    static void *d1=getenv("D1");
+    while (d1) d1=d1;
```

Comment:
This throws an error with clang:
    error: explicitly assigning value of variable of type 'void *' to itself [-Werror,-Wself-assign]
        while (d1) d1=d1;
Do you still need this code?